extensions/cornerstone-dicom-sr/src/tools/DICOMSRDisplayTool.ts

```diff
@@ -48,8 +48,8 @@ export default class DICOMSRDisplayTool extends AnnotationTool {
     const { element } = viewport;
 
     let annotations = annotation.state.getAnnotations(
-      element,
-      this.getToolName()
+      this.getToolName(),
+      element
```

Comment:
Doesn't it take the annotationGroupId now?

Reply:
No, it is the element, the annotation manager decides on the groupId based on the provided element, element can be a string (which will act as the groupId)